src/DataObjects/QueuedJobDescriptor.php

```diff
@@ -389,188 +392,214 @@ public function getJobStatusValues(): array
-                    $this->StepsProcessed,
-                    $this->TotalSteps,
-                    $this->TotalSteps > 0 ? ($this->StepsProcessed / $this->TotalSteps) * 100 : 0
-                )
-            ),
-            $jobTitle = TextField::create('JobTitle', 'Title'),
-            $status = $this->buildJobStatusField(),
-            $jobType = $this->buildJobTypeField(),
-            $runAs,
-            $startAfter = DatetimeField::create('StartAfter', 'Scheduled Start Time'),
-            HeaderField::create('JobTimelineTitle', 'Timeline'),
-            LiteralField::create(
-                'JobTimelineIntro',
-                sprintf(
-                    '<p>%s</p>',
-                    'It is recommended to avoid editing these fields'
-                    . ' as they are managed by the Queue Runner / Service.'
-                )
-            ),
-            $jobStarted = DatetimeField::create('JobStarted', 'Started (initial)'),
-            $jobRestarted = DatetimeField::create('JobRestarted', 'Started (recent)'),
-            $jobFinished = DatetimeField::create('JobFinished', 'Completed'),
-        ]);
-
-        $jobFinished->setDescription('Job completion time.');
-        $jobRestarted->setDescription('Most recent attempt to run the job.');
-        $jobStarted->setDescription('First attempt to run the job.');
-        $jobType->setDescription('Type of Queue which the jobs belongs to.');
-        $status->setDescription('Represents current state within the job lifecycle.');
-
-        $jobTitle->setDescription(
-            'This field can be used to hold user comments about specific jobs (no functional impact).'
-        );
+        $this->beforeUpdateCMSFields(function (FieldList $fields) {
+            $runAs = $fields->fieldByName('Root.Main.RunAsID');
+            $fields->removeByName([
+                'Expiry',
+                'Implementation',
+                'JobTitle',
+                'JobFinished',
+                'JobRestarted',
+                'JobType',
+                'JobStarted',
+                'JobStatus',
+                'LastProcessedCount',
+                'NotifiedBroken',
+                'ResumeCounts',
+                'RetryCount',
+                'RunAs',
+                'RunAsID',
+                'SavedJobData',
+                'SavedJobMessages',
+                'Signature',
+                'StepsProcessed',
+                'StartAfter',
+                'TotalSteps',
+                'Worker',
+                'WorkerCount',
+            ]);
 
-        $startAfter->setDescription(
-            'Used to prevent the job from starting earlier than the specified time.'
-            . ' Note that this does not guarantee that the job will start'
-            . ' exactly at the specified time (it will start the next time the cron job runs).'
-        );
+            // Main
+            $fields->addFieldsToTab('Root.Main', [
+                CompositeField::create([
```

Comment:
Added bunch of composite fields to visually group existing fields together (no change of order of fields)

Reply:
Unless that's specifically related to the automatic retries, I'd prefer this be done separately so the two unrelated changes don't have to wait for one another.